src/monster.cpp

```diff
@@ -3589,3 +3595,8 @@ std::set<tripoint> monster::get_path_avoid() const
 {
     return std::set<tripoint>();
 }
+
+std::vector<std::string> monster::get_trap_avoids() const
```

Comment:
```suggestion
const std::set<trap_id> &monster::get_trap_avoids() const
```

Reply:
adding function to a popular class header whose only user is the class's own implementation, and it only saves typing like 7 characters seems not really necessary

Reply:
I wanted to keep the code similar to what we already have such as get_pathfinding_settings etc.

Reply:
Should I instead directly reference the attribute by trap->trap_avoids instead of the getter function?

Reply:
get_pathfinding_settings is abstract function from Creature and monster class must implement it, it's used in many places and has potential to be used even more as code evolves; while this field isn't "owned" by monster to begin with so it's at best a "nice shortcut"; but it's used only once, inside the class itself and very little potential for other classes being interested in it (and if they are they can just grab it via mtype)

Reply:
Removed superfluous getter function.